RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -494,16 +661,16 @@ vtkSlicerRoomsEyeViewModuleLogic::LoadTreatmentMachine(vtkMRMLRoomsEyeViewNode*
   FILE *fp = fopen(descriptorFilePath.c_str(), "r");
   if (!fp)
     {
-    vtkErrorMacro("LoadTreatmentMachine: Failed to load treatment machine descriptor file '" << descriptorFilePath << "'");
-    return std::vector<TreatmentMachinePartType>();
+      vtkErrorMacro("LoadTreatmentMachine: Failed to load treatment machine descriptor file '" << descriptorFilePath << "'");

```

Comment:
It's not here where you need to add spaces. It is the `{` from where you need to delete two spaces.